schemas/semconv.schema.json

```diff
@@ -99,6 +106,44 @@
 				}
 			]
 		},
+		"EntitySemanticConvention": {
+			"allOf": [
+				{
+					"$ref": "#/$defs/SemanticConventionBase"
+				}
+			],
+			"required": [
+				"name"
+			],
+			"properties": {
+				"type": {
+					"type": "string",
+					"const": "entity"
+				},
+				"attributes": {
+					"type": "array",
+					"items": {
+						"$ref": "#/$defs/Attribute"
+					},
+					"description": "list of identifying attributes for the entity type"
+				},
+				"descriptive_attributes": {
```

Comment:
This doesn't exist yet.

Also I'm going to be talking about this in today's tooling sync, but we may actually want descriptive to be an aspect of attribute, vs. a separate bundle, both for the `ref` aspect of weaver and the `diff` aspect.